infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -2,17 +2,18 @@
 x-defaults: &default-release-image-source
   releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
   releaseImageName: dpl-cms-source
-  dpl-cms-release: "2025.36.1"
-  go-release: 2025.36.2
+  dpl-cms-release: 2025.38.0
+  go-release: 2025.38.0
   php-version: 8.1
 x-webmasters-on-weekly-release-cycle: &webmasters-on-weekly-release-cycle
   #This release cycle releases the newest release, which has been tested on webmaster moduletests in the previous week, on to production
   releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
   releaseImageName: dpl-cms-source
-  dpl-cms-release: "2025.34.0"
-  moduletest-dpl-cms-release: "2025.36.1"
-  go-release: 2025.34.1
+  dpl-cms-release: "2025.36.1"
+  moduletest-dpl-cms-release: 2025.38.0
+  go-release: 2025.36.2
   php-version: 8.1
```

Comment:
Add a `#todo` here, so we remember to bump it with next release

Reply:
done